### PR TITLE
Db 319 error data

### PR DESCRIPTION
**Pull request by jworcestBAH**
@bsweger, @nmonga91: Database changes needed for DB-319, and functionality to report the error type as either header, row-level, or none.  Dependent on the datetime_refactor PR.

### CODE REVIEW

File: dataactcore/migrations/versions/a87f915e4b62_datetimerefactor.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+
+
+
+def upgrade_error_data():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_column('file_status', 'row_errors_present')
```
Comment (bsweger) [inline dataactcore/migrations/versions/a87f915e4b62_datetimerefactor.py]: This line throws an error when the Alembic scripts run b/c this migration occurs before row_errors_present is added to the file_status table.

**Comment by bsweger** [comment]
@jworcestBAH done reviewing

File: dataactcore/migrations/versions/a87f915e4b62_datetimerefactor.py (hunk 1 above)
Comment (jworcestBAH) [inline dataactcore/migrations/versions/a87f915e4b62_datetimerefactor.py]: Definitely something strange going on here, we shouldn't have that column dropped at any point, trying to figure out the current alembic chain again

Comment (jworcestBAH) [inline dataactcore/migrations/versions/a87f915e4b62_datetimerefactor.py]: Okay I think the migrations were out of order when this was generated, and when the order was rearranged I didn't catch this at that point.  I've removed these lines now and verified that the current order is handling row_errors_present correctly.

Comment (bsweger) [inline dataactcore/migrations/versions/a87f915e4b62_datetimerefactor.py]: Thanks--got the migrations to run now.

**Pull request merged**